openedx/features/offline_mode/constants.py

```diff
@@ -9,7 +9,7 @@
 MATHJAX_CDN_URL = f'https://cdn.jsdelivr.net/npm/mathjax@{MATHJAX_VERSION}/MathJax.js'
 MATHJAX_STATIC_PATH = os.path.join('assets', 'js', f'MathJax-{MATHJAX_VERSION}.js')
 
-DEFAULT_OFFLINE_SUPPORTED_XBLOCKS = ['html']
+DEFAULT_OFFLINE_SUPPORTED_XBLOCKS = ['html', 'problem']
```

Comment:
This was fixed in the my [PR](https://github.com/raccoongang/edx-platform/pull/2579)

Reply:
Yes, I know. I think this would not cause conflicts. But, I will remove this change